### ISSUE TITLE
Cannot upload SVGs to image fields (again).

### ISSUE THREAD

**Issue by seantjohnson-dev** [presscodes/kirki#2433]
### Issue description:

This issue seems to have arisen again ever since the image control's 
```
'choices' => [
  'save_as' => ['url', 'id', 'width', 'height']
]
```
changed to 

```
'choices' => [
  'save_as' => 'array'
]
```

I originally opened this issue #1883 that was fixed in [070f27e](https://github.com/kirki-framework/kirki/commit/070f27e17df851daf978d15e9e2347bb5186b83f).

The issue comes from assuming `jsonImg.sizes` always exists, which it does not when SVG's are uploaded. The offending line number is 66 in [packages/kirki-framework/control-image/src/control.js](https://github.com/kirki-framework/kirki/blob/master/packages/kirki-framework/control-image/src/control.js)

### Version used:
4.0.8

(Did you try using the develop branch from GitHub? There's a chance your issue has already been addressed there)

Yes. The issue still exists in the develop branch.

### Using theme_mods or options?
theme_mods

### Code to reproduce the issue (config + field(s))

Changing 
```
url: jsonImg.sizes.full.url,
```
to
```
url: ( ! _.isUndefined( jsonImg.sizes ) ) ? jsonImg.sizes.full.url : jsonImg.url,
```
solves the issue. See below.

Original:

```
if ( 'array' === saveAs ) {
        control.setting.set( {
		id: jsonImg.id,
		url: jsonImg.sizes.full.url,
		width: jsonImg.width,
		height: jsonImg.height
	} );
} else if ( "id" === saveAs ) {
	control.setting.set( jsonImg.id );
} else {
	control.setting.set( ( ! _.isUndefined( jsonImg.sizes ) ) ? jsonImg.sizes.full.url : jsonImg.url );
}
```

Updated:

```
if ( 'array' === saveAs ) {
	control.setting.set( {
		id: jsonImg.id,
		url: ( ! _.isUndefined( jsonImg.sizes ) ) ? jsonImg.sizes.full.url : jsonImg.url,
		width: jsonImg.width,
		height: jsonImg.height
	} );
} else if ( 'id' === saveAs ) {
	control.setting.set( jsonImg.id );
} else {
	control.setting.set( ( ! _.isUndefined( jsonImg.sizes ) ) ? jsonImg.sizes.full.url : jsonImg.url );
}
```

**Comment by contactjavas** [comment]
Thanks for reporting this @seantjohnson-dev !
We'll check this soon. Also, thanks for the codes!

**Comment by contactjavas** [comment]
Hi @seantjohnson-dev , thanks for the fix. It has been included in `v4.0.12`
Download link: [v4.0.12](https://github.com/kirki-framework/kirki/files/7847745/kirki.zip)

Please let us know if that also works on your side :)
Thanks!
